### PR TITLE
3 prototype spotify playback sdk

### PR DESCRIPTION
**Pull request by EddieTheCubeHead**
A functional prototype that logs in to spotify with OAuth and then uses existing playback device to play "Never gonna give you up" by Rick Astley

### CODE REVIEW

**Review by tuomasmustakallio — APPROVED** [review]
LGTM

**Review by zahidhussaina2l — APPROVED** [review]

**Review by ddevarsh7 — APPROVED** [review]

**Review by OttoAhlfors — APPROVED** [review]

**Review by KeManen — APPROVED** [review]

File: poc/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+secrets.json
```
Comment (BadhonRahman01) [inline poc/.gitignore:1]: Endline missing!!

**Review by BadhonRahman01 — APPROVED** [review]
Minor: Only one file endline missing, Except that everything looks good to me.

File: poc/.gitignore (hunk 1 above)
Comment (EddieTheCubeHead) [reply poc/.gitignore:1]: Eh, proof-of-concept folder. We'll be more careful with these in the main code.

**Pull request merged**